### ISSUE TITLE
Jackett support broken? 

### ISSUE THREAD

**Issue by FirstLobster** [erickok/transdroid-search#100]
Can't get the rss search to work.
This is my string. 
http://192.168.1.25:9117/jackett/api/v2.0/indexers/kickasstorrent/results/torznab/api?apikey=APIKEY&t=search&cat=&q=%s
If i paste it in a web browser it return hits. 
I've tried different jackett feeds with the same result.

**Comment by linost-xx** [comment]
I have the same issue.

If I use the URL in Transdroids internal RSS feed with a static search string (no %s) transdroid can display the results. But when I use it in torrent search (with %s) I get "No results for your query"

Is there any logging for this? I don't see the request on the server side.

I also use http, is https a requirement?

**Comment by magic-flapping-pony** [comment]
TL;DR:
**You have to use HTTPS.**
**Workaround for HTTP: build with android:usesCleartextTraffic="true" added to AndroidManifest.xml.** _[Yes, its a bit dirty]_

In detail:
I had the same issue, so I started digging.
When I started debugging the code, I saw that the issue comes from the RssParser parse() function:

![transdroid-search-issue-small](https://user-images.githubusercontent.com/74935195/114306560-6d2bc600-9adc-11eb-824a-d022e0e11c49.png)

The reason seems to be that _"Starting with Android 9 (API level 28), cleartext support is disabled by default."_
(see more at https://stackoverflow.com/questions/45940861/android-8-cleartext-http-traffic-not-permitted)

As a quick workaround, I added _android:usesCleartextTraffic="true"_ to _AndroidManifest.xml_.
With this, the Jackett queries worked like a charm.

There must be some difference between the implementation in transdroid and transdroid-search, because as @FirstLobster mentioned, the static queries using HTTP work there.

**Comment by blackCauldron7** [comment]
@magic-flapping-pony can i get a working apk?
